src/pages/about.svelte

```diff
@@ -115,6 +115,20 @@
 </Container>
 <Container>
 	<h2>Changelog</h2>
+	<Container>
+		<h2>1.9.0</h2>
+		<ul>
+			<li>Add api accounts support</li>
```

Comment:
Change this to "Migrate account actions to API requests" or "Use API requests for account actions"

Reply:
We’ve been calling these changes api accounts everywhere else but I’ll change that

Reply:
> We’ve been calling these changes api accounts everywhere else but I’ll change that

Yeah it just seems off, and most definitely confusing to less technical users